documentation/src/main/asciidoc/topics/code_examples/AuthorMarshaller.java

```diff
@@ -13,7 +13,7 @@ public String getTypeName() {
    }
 
    @Override
-   public void writeTo(ProtoStreamWriter writer, Author author) throws IOException {
+   public void writeTo(MessageMarshaller.ProtoStreamWriter writer, Author author) throws IOException {
```

Comment:
@ryanemerson Can you please double check this change? There are a few other changes to the marshalling content.